shared/src/business/entities/Stamp.test.js

```diff
@@ -0,0 +1,63 @@
+const { formatNow } = require('../utilities/DateHandler');
+const { MOTION_STATUSES } = require('./EntityConstants');
+const { Stamp } = require('./Stamp');
+
+const { VALIDATION_ERROR_MESSAGES } = Stamp;
+
+describe('Stamp entity', () => {
+  describe('Validation', () => {
+    it('should be invalid when status is undefined', () => {
+      const stamp = new Stamp({});
+
+      expect(stamp.getFormattedValidationErrors()).toMatchObject({
+        status: VALIDATION_ERROR_MESSAGES.status,
+      });
+    });
+
+    it('should be invalid when date is in the past', () => {
+      const stamp = new Stamp({
+        date: '2019-12-05T00:00:00.000-05:00',
+        dueDateMessage: 'something',
+        status: MOTION_STATUSES.GRANTED,
+      });
```

Comment:
I would prefer to calculate this date so that we are checking the latest invalid date.
```suggestion
  it('should be invalid when date is yesterday', () => {
      const yesterdayIso = calculateISODate({ howMuch: -1, units: 'days' });
      const yesterdayFormatted = formatDateString(
        createISODateString(yesterdayIso),
        FORMATS.ISO,
      );
      const stamp = new Stamp({
        date: yesterdayFormatted,
        dueDateMessage: 'something',
        status: MOTION_STATUSES.GRANTED,
      });
```